students/14th/youngjoonPark/js/login.js

```diff
@@ -0,0 +1,19 @@
+const button = document.querySelector('.login-btn');
+const id = document.querySelector('.id-box');
+const pw = document.querySelector('.pw-box');
+
+const changeButtonColor = () => {
+  if (id.value.length && pw.value.length) {
+    button.style.backgroundColor = '#0095F6';
+  } else {
+    button.style.backgroundColor = '#BFE0FD';
+  }
+};
```

Comment:
조건에 따라서 로직 잘 처리해주셨네요! 아래와 같이 삼항 연산자로도 할 수 있습니다 :) 검색해보세요!
```suggestion
const changeButtonColor = () => {
  const isUserInfoValid = id.value.length && pw.value.length;
  button.style.backgroundColor = isUserInfoValid ? '#0095F6' : '#BFE0FD';
};
```

Reply:
흠...삼항연산자 개념을 잘 모르겠네요ㅠ 검색해보겠습니다!